crates/models/src/captures.rs

```diff
@@ -55,7 +55,7 @@ pub struct CaptureBinding {
     // Note(johnny): If we need to add details about how data is written to a
     // target, we should turn this into a Target enum as has already been done
     // with Source (used by Materialization & Derive).
-    pub target: Collection,
+    pub target: Option<Collection>,
```

Comment:
I'm glad you mentioned this, because I agree that an explicit `null` is better, and I don't think this implementation is quite right because `target` ought to at least be required in the json schema. I'll figure out where that went wrong and make sure it's required.